docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -23,13 +122,57 @@ creating a new {sn-itsm} connector or
 
 . Install
 https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88[Elastic for ITSM]
-from the {sn} Store.
+from the {sn} store.
+. <<servicenow-itsm-connector-privileges,Assign cross-scope privileges for the Elastic for ITSM app>>.
 . <<servicenow-itsm-connector-prerequisites-integration-user,Create a {sn} integration user and assign it the appropriate roles>>.
 . <<servicenow-itsm-connector-prerequisites-cors-rule,Create a Cross-Origin Resource Sharing (CORS) rule>>.
 . If you use open authorization (OAuth), you must also:
 .. <<servicenow-itsm-connector-prerequisites-rsa-key,Create an RSA keypair and add an X.509 Certificate>>.
 .. <<servicenow-itsm-connector-prerequisites-endpoint,Create an OAuth JWT API endpoint for external clients with a JWT Verifiers Map>>.
 
+[float]
+[[servicenow-itsm-connector-privileges]]
+==== Assign cross-scope privileges
+
+The Elastic for ITSM app requires specific cross-scope privilege records to run successfully:
```

Comment:
Are we going to do the same for SecOps on a different PR?

Reply:
That's right.